shared/js/background/load.es6.js

```diff
@@ -14,32 +14,14 @@ function url (url) {
     return loadExtensionFile({ url: url, source: 'external' })
 }
 
-function returnResponse (xhr, returnType) {
-    if (returnType === 'json' && xhr && xhr.responseText) {
-        let res
-
-        try {
-            res = JSON.parse(xhr.responseText)
-        } catch (e) {
-            console.warn(`couldn't parse JSON response: ${xhr.responseText}`)
-        }
-
-        return res
-    } else if (returnType === 'xml') {
-        return xhr.responseXML
-    } else {
-        return xhr.responseText
-    }
-}
-
 /*
  * Params:
  *  - url: request URL
  *  - source: requests are internal by default. set source to 'external' for non-extension URLs
  *  - etag: set an if-none-match header
  */
 function loadExtensionFile (params) {
-    let xhr = new XMLHttpRequest()
+    const headers = new Headers()
```

Comment:
XMLHttpRequest is not available in service worker, which forces us (thank goodness) to use `fetch`